articles/api-auth/tutorials/authorization-code-grant-pkce.md

```diff
@@ -12,6 +12,15 @@ This tutorial will help you implement the Authorization Code (PKCE) grant. If yo
 
 The __Authorization Code with PKCE__ is the OAuth 2.0 grant that [native apps](/quickstart/native) use in order to access an API. In this document we will work through the steps needed in order to implement this: create a code verifier and a code challenge, get the user's authorization, get a token and access the API using the token.
 
+::: panel Prerequisites
+
+Before beginning this tutorial, please:
+
+* Set the Client's [Grant Type property](/clients/client-grant-types)
+* [Register the API](/apis#how-to-configure-an-api-in-auth0) with Auth0
+* Set the [Default Audience and/or Default Directory](/dashboard-account-settings#api-authorization-settings) 
```

Comment:
I'm wondering whether this last bullet should be there (and all other docs in this PR). It's not actually a prerequisite, it's something that the users can do optionally. You could convert this panel to a paragraph and add a note saying that optionally they can set this and that.